### ISSUE TITLE
Header -O problem with -b beautify and -f format

### ISSUE THREAD

**Issue by psb777** [harelba/q#309]
If one adjusts an output column width using -f then -b does not make the same adjustment with the header row as it does with the data row.

**Comment by harelba** [comment]
True, I've just noticed this as well. I'll fix for the next release, thanks for reporting it.
